src/dotnet/CarbonAware.WebApi/api.yaml

```diff
@@ -0,0 +1,180 @@
+openapi: 3.0.0
+info:
+  description: 'Carbon Aware REST API'
+  version: 0.1.0
+  title: Carbon Aware REST API
+
+paths:
+  '/sci-scores':
+    post:
+      summary: Calculate the SCI score using the configured plugins
+      description: ''
+      operationId: calculateSciScore
+      requestBody:
+        '$ref': '#/components/requestBodies/MarginalCarbonIntensityCalculation'
+        examples: 
+          sci-score-calculation:
+            '$ref': '#/components/examples/sci-score-calc-example'
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                '$ref': '#/components/schemas/SciScore'
+              examples: 
+                sci-score-response:
+                  '$ref': '#/components/examples/sci-score-response-example'
+        '400':
+          description: bad request
+          content:
+            application/json:
+              schema:
+                '$ref': '#/components/schemas/CarbonAwareError'
+  '/sci-scores/marginal-carbon-intensity':
+    post:
+      summary: Calculate the average marginal carbon intensity for a given location and time duration.
+      description: ''
+      operationId: calculateMarginalCarbonIntensity
+      requestBody:
+        '$ref': '#/components/requestBodies/MarginalCarbonIntensityCalculation'
```

Comment:
I'm not sure why, but when I use https://editor.swagger.io/ to view this YAML, I get the following error:

```
$ref and Sibling Elements Any sibling elements of a $ref are ignored
```

I think this might be causing the YAML to display incorrectly.